src/formatting/formatter.rs

```diff
@@ -616,7 +615,7 @@ mod tests {
                 &config,
             )
             .unwrap();
-        assert_eq!(result, "999B");
+        assert_eq!(result, "1.0KB");
```

Comment:
What are the rules if this should be `" 1KB"` or `"1.0KB"`? Try to eliminate trailing zeros?

Reply:
Actually `1.0KB` is consistent with these formats:

Format | Value
-- | --
`$mem_total.eng(p:Mi,w:5)` | `14.96GiB`
`$mem_total.eng(p:Mi,w:4)` | `15.0GiB`
`$mem_total.eng(p:Mi,w:3)` | ` 15GiB`
`$mem_total.eng(p:Mi,w:2)` | `15GiB`

So I think this is good as-is.